src/main/java/mate/academy/model/Movie.java

```diff
@@ -1,10 +1,10 @@
 package mate.academy.model;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 
```

Comment:
why are you replacing this?

Reply:
It's because of pom.xml dependencies, I'll change it back